bokehjs/src/coffee/models/widgets/markup.coffee

```diff
@@ -24,8 +24,8 @@ class Markup extends Widget.Model
 
   @define {
     text: [ p.String, '' ]
-    width: [ p.Number, 500]
-    height: [ p.Number, 400]
+    width: [ p.Number, none]
+    height: [ p.Number, none]
```

Comment:
There is no `none` on the JS side, you can just omit the second element completely and it will default to `null` (which is what is correct)